apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/context/tag/Tags.java

```diff
@@ -107,6 +107,28 @@ public static final class HTTP {
     }
 
     public static final StringTag LOGIC_ENDPOINT = new StringTag(12, "x-le");
+    /**
+     * CACHE_TYPE records cache type, such as jedis
+     */
+    public static final StringTag CACHE_TYPE = new StringTag(15, "cache.type");
+
+    /**
+     * CACHE_OP represent a command is used for "write" or "read"
+     * It's better that adding this tag to span , so OAP would analysis write/read metric accurately
+     * Reference org.apache.skywalking.apm.plugin.jedis.v4.AbstractConnectionInterceptor#parseOperation
+     * BTW "op" means Operation
+     */
+    public static final StringTag CACHE_OP = new StringTag(16, "cache.op");
+
+    /**
+     * CACHE_TYPE records the cache command
+     */
+    public static final StringTag CACHE_CMD = new StringTag(17, "cache.cmd");
+
+    /**
+     * CACHE_TYPE records the cache key
+     */
+    public static final StringTag CACHE_KEY = new StringTag(18, "cache.key");
```

Comment:
@apache/skywalking-committers Please notice, these new tags would be significant for virtual cache analysis, which would be added in 9.3.0.

If you want your cache framework relative plugins would work like these Java plugin, please follow the tag format protocol.